configs/tests/res50_mpii_256x256.py

```diff
@@ -0,0 +1,94 @@
+_base_ = [
```

Comment:
Please move the verified configs to proper locations like `configs/body2d_keypoints/topdown_regression/mpii/'.